src/client/lcd/Wallet.ts

```diff
@@ -89,7 +89,9 @@ export class Wallet {
         chainID: options.chainID,
         signMode:
           options.signMode ||
-          (this.lcd.config[options.chainID].isClassic
+          (options.msgs.find(m => 'aminoOnly' in m && m.aminoOnly)
```

Comment:
You could check the type by using instanceof to avoid adding a boolean to the class. What do you think?
```suggestion
          (options.msgs.find(m => m instanceof MsgAminoCustom)
```

> if decide to apply this change you may have to remove aminoOnly property from the object and import MsgAminoCustom into this file